.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+<!-- IMPORTANT!
+  - Every PR must reference an issue; this helps to explain the intent of the PR
+ -->
+
+Fixes #
+
+# 👩🏻‍💻 What does this PR do? 
+ <!-- Explain the changes you made, and why they're needed. Add a screenshot if you've made any UI changes!  -->
+
+# 🧪 How has this change been tested? 
+<!-- Explain how to setup for testing here if it is not already obvious, and how you've tested this PR. -->
+
+## 👀 Related areas to think about 
+<!-- If there are any general areas of the codebase your changes might have have touched or could cause side effects to, mention them here --> 
+
+## 💌 Any notes for the reviewer?
+<!-- eg. Do you have any specific questions for the reviewer? Is there a high risk/complicated change they should focus on? -->
+
+## 📃 Documentation
+<!-- Note down any areas which require documentation updates -->
+_No user facing changes_
```

Comment:
Shouldn't you either include the docs in PR or write an issue to do it later?

Reply:
Something that's worked reasonably well in mSupply Desktop is having labels for different doc requirements; _not needed_, _internal_ and/or _public_. Easy to find issues that require doc updates by category later.

For instance, some sync changes might want internal doc changes. A bug fix may just be aligning the system with the behaviour in public docs, so no change needed. Changing a UI element might need images throughout docs to be updated (you might want to note this kind of thing in this "Documentation" section of the template I figure).

Reply:
You should include docs, that's true.
I was thinking of the public facing user documentation - and flagging whether this PR introduces changes that necessitate changes or additions to the docs. Updates to the readmes are a different story!